package.json

```diff
@@ -68,5 +69,5 @@
   "bugs": {
     "url": "https://github.com/MauricioRobayo/react-obfuscate-email/issues"
   },
-  "homepage": "https://www.mauriciorobayo/react-obfuscate-email"
+  "homepage": "https://www.mauriciorobayo.com/react-obfuscate-email"
```

Comment:
@scan-lan Could you please revert this line?

Reply:
Done :)